.gitignore

```diff
@@ -7,4 +7,4 @@
 /version.txt
 .idea/
 cmake-build*/
-
+/CMakeLists.txt.user
```

Comment:
Could you put newlines at the end of these files? Thanks.

Reply:
Done